chrome/skin/default/zotero/itemPane.css

```diff
@@ -42,13 +42,13 @@
 }
 
 /* Restore/Delete buttons in trash view */
-#zotero-item-top-buttons-holder {
+#zotero-item-pane-top-buttons-trash {
 	-moz-appearance: toolbar;
 	-moz-box-pack: center;
 	min-height: 3em;
 }
 
-#zotero-item-top-buttons-holder > button {
+#zotero-item-pane-top-buttons-trash > button {
 	-moz-box-flex: 1
 }
 
```

Comment:
The feed buttons need these styles too — should use a class that applies to both.